src/Xamarin.Android.Build.Tasks/Utilities/ZipArchiveEx.cs

```diff
@@ -124,6 +124,22 @@ public void RemoveFile (string folder, string file)
 				zip.DeleteEntry ((ulong)index);
 		}
 
+		public bool MoveEntry (string from, string to)
+		{
+			if (!zip.ContainsEntry (from)) {
+				return false;
+			}
+			var entry = zip.ReadEntry (from);
+			using (var stream = new MemoryStream ()) {
+				entry.Extract (stream);
+				stream.Position = 0;
+				zip.AddEntry (to, stream);
```

Comment:
What will `zip.AddEntry()` do if `to` already exists?  Throw an exception?  Replace it?  Other?

Reply:
This is not a problem. LibZipShrap will override existing items by default.